### DISCUSSION TITLE
Welcome to bldc_motor_controller_pcb Discussions!

### DISCUSSION THREAD

**Discussion by azmat-bilal** [azmat-bilal/bldc_motor_controller_pcb#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sergeytlt** [comment]
Hey, here is 0 comments, does this hardware works well? Which firmware from Odrive did you use v3.5 or 3.4?
